help/sources/tutorials/api/create/customer-success/zendesk.md

```diff
@@ -0,0 +1,1159 @@
+---
+keywords: Experience Platform;home;popular topics;sources;connectors;source connectors;sources sdk;sdk;SDK
+solution: Experience Platform
+title: Documentation self-service template
+topic-legacy: tutorial
+description: Learn how to connect Adobe Experience Platform to Zendesk using the Flow Service API.
+hide: true
+hidefromtoc: true
+exl-id: 
+---
+# Create a Zendesk connection using the Flow Service API
+
+## Overview
+
+[Zendesk](https://www.zendesk.com) is a popular customer service solution and sales tool.
+
+This [!DNL Adobe Experience Platform] [!DNL sources connector](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) leverages the [!DNL Zendesk Search API > Export Search Results](https://developer.zendesk.com/api-reference/ticketing/ticket-management/search/#export-search-results) that returns users information into Experience Platform from Zendesk for further processing.
```

Comment:
We can remove the three DNL tags in this paragraph and let's reword "sources" to just "source"